internal/core/items.rs

```diff
@@ -502,7 +522,8 @@ impl ItemRc {
             },
             &|root| root.next_sibling(),
             &|start, _| start,
-            &|root| root,
+            &std::convert::identity,
```

Comment:
Same fix for the build as when this initially landed:

```suggestion
            &core::convert::identity,
```